src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -142,6 +143,25 @@ internal class OktaApi : IOktaApi {
 		throw new BmxException( "Account could not be found" );
 	}
 
+	async Task<string> IOktaApi.GetMeResponseAsync( string sessionId ) {
+
+		try {
+			using var meResponse = await _httpClient.GetAsync( "users/me" );
+
+			if( !meResponse.IsSuccessStatusCode ) {
+				return "";
```

Comment:
similarly here (and below), I'd return null rather than an empty string.
the signature of the method could be
```csharp
Task<string?> IOktaApi.GetMeUserIdOrNullAsync( string sessionId )
```
which better illustrate the behaviours of this method.

Reply:
When you return empty strings, the method signature doesn't communicate that behaviour to the caller. The caller can't know that they should check for empty strings unless they read the docs or code of your method.

When you return nulls, the caller would be forced to check for nulls, so this is less error prone and a better dev experience.